extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -65,6 +66,41 @@ const ConversionRate = ({
   </div>
 );
 
+const BalanceOption = ({
```

Comment:
tried to make this pretty standalone as I can see this being used elsewhere later